src/NServiceBus.Gateway.AcceptanceTests/EndpointCustomizationConfigurationExtensions.cs

```diff
@@ -23,7 +24,7 @@ public static IEnumerable<Type> GetTypesScopedByTestClass(this EndpointCustomiza
 
             types = types.Union(endpointConfiguration.TypesToInclude);
 
-            return types.Where(t => !endpointConfiguration.TypesToExclude.Contains(t)).ToList();
+            return types.Where(t => !t.IsAssignableTo(typeof(INeedToInstallSomething)) && !endpointConfiguration.TypesToExclude.Contains(t)).ToList();
```

Comment:
Does it mean we also need to exclude features here because we have these redundant extensions?